src/server/config.js

```diff
@@ -1,51 +1,49 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 
-var fs = require('fs'),
-    path = require('path');
-
-var config = {};
+/**
+ * @constructor
+ */
+function Configuration() {
+    this._config = {};
+}
 
-// module properties
 [
     { name: 'debugHostHandlers', optional: true },
     { name: 'forcePrepare' },
     { name: 'liveReload' },
     { name: 'platform' },
-    { name: 'platformRoot' },
-    { name: 'projectRoot', single: true },
-    { name: 'server', optional: true },
+    // NOTE: platformRoot and projectRoot belongs to the Project instance
+    // { name: 'platformRoot' },
+    // { name: 'projectRoot', single: true },
+    // { name: 'server', optional: true },
```

Comment:
I leave these lines commented just for information about the changes done, but it should be removed.